apps/rush/typings/encode-registry/encode-registry.d.ts

```diff
@@ -0,0 +1,6 @@
+// typings for encode-registry@1.0.0
+
+declare module 'encode-registry' {
+  function encode(registry: string): string;
+  export = encode;
+}
```

Comment:
Submit back to definitelyTyped?

Reply:
We can do this later

---
In reply to: [148673120](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r148673120) [](ancestors = 148673120)